src/sentry/models/useroption.py

```diff
@@ -116,6 +116,42 @@ class UserOption(Model):
     Keeping user feature state
     key: "feature:assignment"
     value: { updated: datetime, state: bool }
+
+    where key is one of:
```

Comment:
I think you should add a note here inviting developers to add their keys and descriptions here if they create a new UserOption.